posteriors/laplace/dense_ggn.py

```diff
@@ -137,7 +137,7 @@ def outer_loss(z, batch):
 
     with torch.no_grad(), CatchAuxError():
         ggn_batch, aux = ggn(
-            partial(forward, batch=batch),
+            lambda params: forward(params, batch),
             partial(outer_loss, batch=batch),
```

Comment:
Think we need to change `outer_loss` to a `lambda` too

Reply:
It's not strictly needed as the `outer_loss` function is defined above `L135` (by the lib) and so kw arguments pass just fine.

Updated to use a lambda. I think having them the same is simpler to read.